recipes-kernel/linux/linux-armv7multi.inc

```diff
@@ -0,0 +1,43 @@
+# custom oe-based kernel include for rcn armv7multi
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+inherit kernel
+require recipes-kernel/linux/linux-dtb.inc
+
+SUMMARY = "Linux kernel"
+SECTION = "kernel"
+LICENSE = "GPLv2"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
+
+INC_PR = "r1"
+
+DEPENDS += "xz-native bc-native"
+RDEPENDS_kernel-base += "kernel-devicetree"
+
+KERNEL_DEVICETREE_nitrogen6x = "imx6q-sabrelite.dtb \
+                  imx6q-nitrogen6x.dtb imx6dl-nitrogen6x.dtb \
+                  imx6q-nitrogen6_max.dtb imx6qp-nitrogen6_max.dtb"
+
```

Comment:
I guess this nitrogen6x is a left over from your configuration.

Reply:
Yup, same mainline kernel/graphics config. It's already ignored but feel free to remove it anyway...